lib/org/chromium/webidl/ast/Iterable.js

```diff
@@ -20,6 +20,11 @@ foam.CLASS({
   ],
 
   methods: [
+    function getName() {
+      var types = [this.type.getName()];
+      if (this.valueType) types.push(this.valueType.getName());
+      return `iterable<${types.join(',')}>`;
+    },
```

Comment:
For iterables, the name will look like its definition. (e.g. `iterable<string>`)